integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -499,16 +513,19 @@ public void testErrorPathDomainMissingWDTBinary() {
 
     // create stage dir for common mount image
```

Comment:
Update any references to "common mount image" to "auxiliary image"

Reply:
fixed